Docs: Update quickstart instructions (#13611)
apache/druid: docs/tutorials/index.md

./bin/start-druid
```

> This brings up instances of ZooKeeper and the Druid services and may use up to 80% of the total available system memory. You may also explicitly specify the total memory for Druid to use by passing a value for the memory parameter, e.g. `./bin/start-druid -m 16g` or `./bin/start-druid --memory 16g`.

Reviewer: ```suggestion
This brings up instances of ZooKeeper and the Druid services and may use up to 80% of the total available system memory. To explicitly set the total memory available to Druid, pass a value for the memory parameter, e.g. `./bin/start-druid -m 16g` or `./bin/start-druid --memory 16g`.
```
